src/Uno.UI/UI/Xaml/UIElement.cs

```diff
@@ -528,6 +530,17 @@ internal static Matrix3x2 GetTransform(UIElement from, UIElement to)
 				return Matrix3x2.Identity;
 			}
 
+#if __SKIA__
+			Matrix4x4.Invert(to?.Visual?.TotalMatrix ?? Matrix4x4.Identity, out var invertedTotalMatrix);
```

Comment:
Question: how does winUI react if on of the `UIElement` is not in the visual tree?

Reply:
It returns a `MatrixTransform` with `Matrix == Identity`.